PowerUp/app/src/main/res/layout-land/activity_memory_match_game.xml

```diff
@@ -0,0 +1,141 @@
+            app:layout_constraintEnd_toStartOf="@+id/txt_time"
+            app:layout_constraintTop_toTopOf="parent" />
+
+        <TextView
+            android:id="@+id/txt_time"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginEnd="@dimen/margin_large"
+            android:layout_marginTop="@dimen/margin_large"
+            android:text="@string/txt_zero"
+           android:textColor="@color/powerup_dark_blue"
+            android:textSize="@dimen/txt_size"
+            android:textStyle="bold"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintTop_toTopOf="parent" />
+
+        <TextView
+            android:id="@+id/txt_label_memory"
+            android:layout_width="@dimen/memory_label_width"
+            android:layout_height="wrap_content"
+            android:layout_marginEnd="@dimen/margin_small"
+            android:layout_marginStart="@dimen/margin_small"
+            android:layout_marginTop="@dimen/margin_large"
+            android:text="@string/memory_label"
+            android:textColor="@color/powerup_dark_blue"
+            android:textSize="@dimen/txt_size_large"
+            android:textStyle="bold"
+            app:layout_constraintEnd_toStartOf="@+id/txt_time_label"
+           app:layout_constraintStart_toEndOf="@+id/img_score"
+           app:layout_constraintTop_toTopOf="parent" />
+
+        <Button
+            android:id="@+id/btn_yes"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginBottom="@dimen/btn_margin_bottom"
+            android:background="@drawable/btn_bg"
+            android:text="@string/yes"
+            android:textColor="@color/powerup_dark_blue"
+            android:textStyle="bold"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintEnd_toStartOf="@+id/img_tile_2" />
+
+        <Button
+            android:id="@+id/btn_no"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginBottom="@dimen/btn_margin_bottom"
+            android:background="@drawable/btn_bg"
+            android:text="@string/no"
+            android:textColor="@color/powerup_dark_blue"
+            android:textStyle="bold"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintStart_toEndOf="@+id/img_tile_2" />
+
+    <ImageView
+        android:id="@+id/btn_start"
+        android:layout_width="@dimen/start_width_memory"
+        android:layout_height="@dimen/start_height_memory"
+        android:layout_marginBottom="@dimen/margin_large"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        android:src="@drawable/start_button" />
+
+    </android.support.constraint.ConstraintLayout>
```

Comment:
This file is not related to this PR.
Changes appear here because of previous unmerged changes.